frontend/src/pages/EnvironmentSettings/EnvironmentSettings.wrap.tsx

```diff
@@ -1,26 +1,5 @@
 import { injectIntl } from 'react-intl';
```

Comment:
I think you can just delete this file and edit the `index.ts` to export default from `./EnvironmentSettings`